### DISCUSSION TITLE
A problem with integrate PyVista nad Tkinter

### DISCUSSION THREAD

**Discussion by Bekbo01** [pyvista/pyvista#1597] [Q&A]
Hello I have a problem with integrate PyVista nad Tkinter. Can anyone help with that problem? 
I create a listbox in tkinter and when we choose element of List opened window of Pyvista OpenGL. And it's working, but when I close the pyvista window my tkinter frame doesn't work.
Anyone have idea about that?
And I try to make like that https://stackoverflow.com/questions/60649673/is-there-a-way-to-integrate-pyvista-with-tkinter but it's same doesn't work.

**Comment by banesullivan** [comment]
I don't know of anyone mixing PyVista with Tkinter (it's def possible) but I'm not sure where to begin with addressing this. The folks over in PyVistaQt might be able to help.... https://github.com/pyvista/pyvistaqt
